tests/Cargo.toml

```diff
@@ -32,4 +32,8 @@ path = "edc_api/edc_api_test.rs"
 
 [[test]]
```

Comment:
If this gets out of hand a dedicated /tests directory would also be a good idea.

Reply:
There is a dedicated /tests folder, but for a better structure and to be able to separate the individual crates of the workspace in the tests, I have subfolders for the individual crates in this test folder, which is why the corresponding structure is created in the Toml.

The /tests folder currently looks like this:
```
    dspaces-rs
    ├── crates
    │   ├── ...
    │   └── ...
    │
    ├── docs
    │   ├── ...
    │   └── ...
    │
    ├── tests
    │   ├── edc_api
    │   │   └──edc_api_test.rs
    │   ├── edc_client
    │   ├── edc_server
    │   ├── odrl
    │   │   ├── json_parser_test.rs
    │   │   └── json_serializer_test.rs
    │   └── cargo.toml
    │
    └── cargo.toml
```